sdk/entra/WebJobs.Extensions.AuthenticationEvents/.gitignore

```diff
@@ -0,0 +1,365 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do you need this file? There is a repo-wide gitignore already - https://github.com/Azure/azure-sdk-for-net/blob/main/.gitignore

Reply:
No longer required. (Was for dev)